package.json

```diff
@@ -45,7 +45,7 @@
     "node": ">=14"
   },
   "dependencies": {
-	  "notion": ""
+    "@notionhq/client": "^2.0.0"
```

Comment:
are we moving to use the notion sdk in the new version?
in that case, replace all. uses of fetch

Reply:
using `fetch` ; the sdk is incomplete it appears